.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+      time: '00:45'
```

Comment:
Is this needed at all? Don't see any GHA nearby.

Reply:
[Yes please](https://github.com/OWASP/www-project-coraza-web-application-firewall/pull/16/files#diff-7baf8649ff890bed0f16535cdeb8bc02e6da81b97b2754f87b8fe4e7de8c2a06R23-R24)